buildSrc/private/src/main/kotlin/androidx/build/LibraryVersionsService.kt

```diff
@@ -63,7 +63,7 @@ abstract class LibraryVersionsService : BuildService<LibraryVersionsService.Para
         val versions = getTable("versions")
         val libsGroupsAndVersions = parameters.libsOverrideVersions.get()
         versions.keySet().associateWith { versionName ->
-            val tagName = libsGroupsAndVersions.keys.firstOrNull { versionName.startsWith(it) }
```

Comment:
It was because groups like `groups.LIFECYCLE_JETBRAINS` inside toml file. It should not work correctly now

Reply:
It compares versions, not groups:
<img width="641" alt="image" src="https://github.com/user-attachments/assets/18d50821-9152-41d3-b658-2bbbce0c04f4">